runtime/container.go

```diff
@@ -37,20 +38,26 @@ type Command interface {
 	Wait() error
 }
 
-func newContainer(c libcontainer.Container, p *Process) Container {
+func newContainer(c libcontainer.Container, p *Process, image *v1.Image) Container {
```

Comment:
`image` is not a Image, is just metainformation should be should change the name to something more descriptive